internal/xds/rbac/converter_test.go

```diff
@@ -110,5 +112,52 @@ func (s) TestBuildLoggerErrors(t *testing.T) {
 
 		})
 	}
+}
 
+func (s) TestBuildLoggerKnownTypes(t *testing.T) {
+	tests := []struct {
+		name         string
+		loggerConfig *v3rbacpb.RBAC_AuditLoggingOptions_AuditLoggerConfig
+	}{
+		{
+			name: "stdout logger",
```

Comment:
So if users send typed structs with the name being `stdout_logger`, we will end up creating the same logger. Can we test those as well?

Reply:
We will accept that and create stdout loggers for those configs. I don't think there is other ways here as authz policy translates the config into typed structs regardless.